test/e2e/flowaggregator_test.go

```diff
@@ -97,10 +97,12 @@ AntreaProxy enabled (Inter-Node): Flow record from destination Node is ignored,
 const (
 	ingressNetworkPolicyName = "test-flow-aggregator-networkpolicy-ingress"
 	egressNetworkPolicyName  = "test-flow-aggregator-networkpolicy-egress"
-	collectorCheckTimeout    = 10 * time.Second
+	collectorCheckTimeout    = 12 * time.Second
+	iperfTime                = 12
 	// Single iperf run results in two connections with separate ports (control connection and actual data connection).
-	// As 5s is export interval and iperf traffic runs for 10s, we expect about 4 records exporting to the flow aggregator.
-	// Since flow aggregator will aggregate records based on 5-tuple connection key, we expect 2 records.
+	// As 2s is the export interval of flow exporter and iperf traffic runs for 12s, we expect about 12 records exporting to the flow aggregator.
+	// Since flow aggregator will aggregate records based on 5-tuple connection key and exports to collector every 4 seconds, we expect 3 records.
+	// However, we could only find two flow records for the iperf flow in Kind clusters, so we set the expected record number to 2 here.
```

Comment:
Can this observation be related to not having bandwidth stats in kind cluster?

Reply:
Tested again and there is no such limitation on my local kind cluster now, removed this comment.